content/token-swap.md

```diff
@@ -0,0 +1,848 @@
+    tokenFeeAccountAddress, // ata
+    feeOwner, // owner
+    poolTokenMint // mint
+)
+
+transaction.add(tokenFeeAccountInstruction)
+```
+
+### Create the swap pool
+
+With all of the prerequisite accounts created, you can create the swap pool initialization instruction using `TokenSwap.createInitSwapInstruction` from the `spl-token-swap` library.
+
+This function takes *a lot* of arguments. Let's talk through them.
+
+The first 7 arguments are the prerequisite token accounts we just discussed.
+
+After that comes the constant representing the Token Program ID followed by the constant representing the Token Swap Program ID.
+
+Next, there are 4 pairs of number arguments representing numerators and denominators for the trade fee, owner trade fee, owner withdraw fee, and host fee. The instruction uses the numerator and denominator for each to calculate the percentage of the fee. Lets explain each of the fees:
+1. **Trade fee** - fees that are retained by the swap pool token accounts during a trade and increase the redeemable value of LP-tokens. This fee rewards users for providing liquidity to the swap pool.
+2. **Owner trade fee** - fees that are retained by the swap pool token accounts during a trade, with the equivalent in LP-tokens minted to the owner of the program
+3. **Owner withdraw fee** - extra LP-tokens that are sent to the owner on every withdrawal
+4. **Host fee** - a proportion of the owner trade fees, sent to an extra host token account provided during the trade. This fee incentives external parties (such as a dex) to provide frontends for the swap pool and rewards them with a portion.
+
+Lastly, there's the curve type, which we'll discuss further later in the lesson.
+
+```tsx
+const createSwapInstruction = TokenSwap.createInitSwapInstruction(
+    tokenSwapStateAccount,      // Token swap account
+    swapAuthority,              // Swap pool authority
+    poolTokenA,                 // Token A token account
+    poolTokenB,                 // Token B token account
+    poolTokenMint,              // Swap pool token mint
+    tokenFeeAccountAddress,     // Token fee account
+    tokenAccountPool.publicKey, // Swap pool token account
+    token.TOKEN_PROGRAM_ID,     // Token Program ID
+    TOKEN_SWAP_PROGRAM_ID,      // Token Swap Program ID
+    0,                          // Trade fee numerator
+    10000,                      // Trade fee denominator
+    5,                          // Owner trade fee numerator
+    10000,                      // Owner trade fee denominator
+    0,                          // Owner withdraw fee numerator
+    0,                          // Owner withdraw fee denominator
+    20,                         // Host fee numerator
+    100,                        // Host fee denominator
+    CurveType.ConstantProduct   // Curve type
+)
+
+transaction.add(createSwapInstruction)
+```
+
+Once the transaction with this instruction successfully executes, the swap pool is ready to be used.
```

Comment:
@jamesrp13 we also start the next sentence with "Once."  I suggest generalizing and changing this sentence opener to:
```
When a transaction with these instructions successfully executes, a swap pool is ready to be used.
```